indico/modules/rb_new/client/js/common/bookings/BookingDetailsModal.jsx

```diff
@@ -0,0 +1,263 @@
+                <Button type="button"
+                        icon="check circle"
+                        color="green"
+                        size="small"
+                        content={<Translate>Accept booking</Translate>} />
+            )}
+        </div>
+    );
+
+    render() {
+        const {
+            booking: {
+                room, bookedForUser, startDt, endDt, repetition, createdByUser, createdDt, bookingReason,
+                occurrences, dateRange, editLogs, canAccept, canCancel, canDelete, canModify, canReject, isCancelled,
+                isRejected, isAccepted
+            }
+        } = this.props;
+        const {occurrencesVisible} = this.state;
+        const dates = {startDate: startDt, endDate: endDt};
+        const createdOn = serializeDate(toMoment(createdDt));
+        const times = {startTime: moment(startDt).utc().format('HH:mm'), endTime: moment(endDt).utc().format('HH:mm')};
+        const recurrence = this.getRecurrence(repetition);
+        const legendLabels = [
+            {label: Translate.string('Occurrence'), color: 'orange'},
+        ];
+        const showActionButtons = (!isCancelled && !isRejected);
+        return (
+            <Modal open onClose={this.handleCloseModal} size="large" closeIcon>
+                <Modal.Header styleName="booking-modal-header">
+                    <Translate>Booking Details</Translate>
+                    <span>
+                        {canModify && <Button icon="pencil" circular />}
+                        {canDelete && <Button icon="trash" circular />}
+                    </span>
+                </Modal.Header>
+                <Modal.Content>
+                    <Grid columns={2}>
+                        <Grid.Column>
+                            <RoomBasicDetails room={room} />
+                            <RoomKeyLocation room={room} />
+                            <TimeInformation recurrence={recurrence}
+                                             dates={dates}
+                                             timeSlot={times}
+                                             onClickOccurrences={() => this.setState({occurrencesVisible: true})}
+                                             occurrencesNumber={dateRange.length} />
+                        </Grid.Column>
+                        <Grid.Column>
+                            {this.renderBookedFor(bookedForUser)}
+                            {this.renderReason(bookingReason)}
+                            {this.renderBookingHistory(editLogs, createdOn, createdByUser)}
+                            {showActionButtons && this.renderActionButtons(canAccept, canCancel, canReject, isAccepted)}
+                        </Grid.Column>
+                    </Grid>
+                </Modal.Content>
+                <Modal open={occurrencesVisible}
+                       onClose={() => this.setState({occurrencesVisible: false})}
+                       size="large" closeIcon>
```

Comment:
i'd put `closeIcon` in a new line, since all other attributes are in their own line as well